Gem.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -74,12 +83,12 @@
         }
       },
       {
-        "package": "WalletConnect",
-        "repositoryURL": "https://github.com/WalletConnect/WalletConnectSwiftV2",
+        "package": "CoinbaseWalletSDK",
```

Comment:
Is it possible to not include `CoinbaseWalletSDK` here?

Reply:
We don't use in the app, it's the dependency listed on reown-swift